### PR TITLE
Add a working Button Class

### PR DESCRIPTION
**Pull request by SamGonChec**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Issue #:
Card Title:

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] App.test
- [ ] Other

**Test Configuration**:
* Dependencies changed:
* Test class name(s):
* Classes tested:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by mumkymikey — APPROVED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Issue #:
+Card Title:
```
Comment (anodyne99) [inline .github/pull_request_template.md:6]: This section confuses me

Comment (anodyne99) [reply .github/pull_request_template.md:6]: Wat dis

**Review by anodyne99 — COMMENTED** [review]
Lorem ipsum dolor

**Review by jvnyxc — APPROVED** [review]

**Review by anodyne99 — APPROVED** [review]
Seems legit

**Review by rpwgc4 — APPROVED** [review]

**Pull request closed without merging**